package.json

```diff
@@ -16,7 +16,7 @@
     "release": "cd dist && npm publish",
     "lint": "tslint 'src/**/*.ts'",
     "preversion": "yarn build && yarn test:dist",
-    "coverage": "jest src --coverage"
+    "coverage": "jest src --coverage --coverageReporters text text-summary"
```

Comment:
Done. And yeah, you are at 34%, so I've bumped lines down to 30% for the meantime.